src/metabase/models/card.clj

```diff
@@ -274,6 +274,49 @@
   pre-update-check-sandbox-constraints
   (atom identity))
 
+(defn- update-parameter-use-card-as-sources
```

Comment:
more grammatically correct, and clearer too?
```suggestion
(defn- update-parameters-using-card-as-values-source
```

Reply:
Renamed in 4b130f5aeb4ec5845af5fdd5be83ebd2095ab62a

Reply:
Did you deliberately make rename it `values-sources` with an `s` at the end? There's one "values source" per parameter, so it should be singular in my book

Reply:
my mistake, changed again.